book/getting-started/run-a-wallaroo-application-docker.md

```diff
@@ -19,12 +19,12 @@ Let's get started!
 
 Since Wallaroo is a distributed application, its components need to run separately, and concurrently, so that they may connect to one another to form the application cluster. For this example, you will need 6 separate terminal shells to start the docker container, run the metrics UI, run a source, run a sink, run the Celsius application, and eventually, to send a cluster shutdown command.
 
-## Shell 1: Start the Wallaroo Docker container
+## Shell 1: Start the Wallaroo Docker container for Machida
 
 {% codetabs name="UNIX Bash", type="bash" -%}
 docker run --rm -it --privileged -p 4000:4000 \
 -v /tmp/wallaroo-docker/wallaroo-{{ book.wallaroo_version }}/wallaroo-src:/src/wallaroo \
--v /tmp/wallaroo-docker/wallaroo-{{ book.wallaroo_version }}/python-virtualenv:/src/python-virtualenv \
+-v /tmp/wallaroo-docker/wallaroo-{{ book.wallaroo_version }}/python-virtualenv-machida:/src/python-virtualenv \
```

Comment:
Should this `machida` specific directory be used for non-`UNIX Bash` environments also?

Reply:
addressed